defaults/main.yml

```diff
@@ -1,9 +1,43 @@
 ---
-portainer_version: 1.11.0
-portainer_port: 5000
-portainer_admin_pass: admin555
-
-swarm_nodes_group: swarm-nodes
-swarm_managers_group: swarm-managers
-
-mode: cluster
+### Portainer version
+version: develop
```

Comment:
I think the default should use a stable release or if we remain with develop branch we should explain why in the README

Reply:
fixed by explaining it in the README and changing to latest